### ISSUE TITLE
storage: sentry: 2.1 alpha cluster repeatedly failing consistency checks

### ISSUE THREAD

**Issue by tbg** [cockroachdb/cockroach#28699]
https://sentry.io/cockroach-labs/cockroachdb/issues/627224998/

```
*log.safeError: replica_consistency.go:154: consistency check failed with %d inconsistent replicas | int
  File "github.com/cockroachdb/cockroach/pkg/storage/replica.go", line 2632, in executeAdminBatch
  File "github.com/cockroachdb/cockroach/pkg/storage/replica.go", line 2000, in sendWithRangeID
  File "github.com/cockroachdb/cockroach/pkg/storage/replica.go", line 1942, in Send
  File "github.com/cockroachdb/cockroach/pkg/storage/store.go", line 3004, in Send
  File "github.com/cockroachdb/cockroach/pkg/storage/stores.go", line 185, in Send
...
(25 additional frame(s) were not displayed)

replica_consistency.go:154: consistency check failed with %d inconsistent replicas | int
```

**Comment by tschottdorf** [comment]
@nstewart why did you change the severities? This issue wouldn't be fixed by an upgrade.

**Comment by tschottdorf** [comment]
@andreimatei I think you had some suggestion about some error type that would make it more likely that people would report this sort of thing? I forgot what exactly it was. My plan would be to make it really easy to open a descriptive issue for this kind of failure.

**Comment by andreimatei** [comment]
See `UnexpectedWithIssueErrorf`.

**Comment by tschottdorf** [comment]
Moving to 2.2. A call to action has been added to fatal messages (#30898).

**Comment by tschottdorf** [comment]
Panic improved in https://github.com/cockroachdb/cockroach/issues/30361. Assumption is that this is disk corruption or other RocksDB-or-below type expected error.

**Comment by tschottdorf** [comment]
Wait, this isn't the right issue. Reopening.

**Comment by tbg** [comment]
This is still happening, but no report from 2.1 (but a few from a 2.1-alpha release). Still feels like we can do more reporting here, for example report the size of the diff and which system keys, if any, it involves.

**Comment by knz** [comment]
Is the S-0 severity still warranted?

**Comment by tbg** [comment]
I'm not seeing anything about 2.1 in the events (there are lots of reports from development versions in the past, which I think is due to some merge bugs triggered in roachtest?)

I've marked the sentry issue as "fixed in v2.1.0" which means that it should reopen if we see a report from v2.1.0 or higher.
